runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -76,6 +77,24 @@ J9::CodeCacheManager::initialize(bool useConsolidatedCache, uint32_t numberOfCod
    return self()->OMR::CodeCacheManager::initialize(useConsolidatedCache, numberOfCodeCachesToCreateAtStartup);
    }
 
+bool
+J9::CodeCacheManager::isAvailableMemorySafeForAllocation(size_t &requestedCodeCacheSize)
+   {
+   bool incomplete;
+   uint64_t freePhysicalMemoryB = getCompilationInfo(_jitConfig)->computeAndCacheFreePhysicalMemory(incomplete);
+
+   if (TR::Options::isAnyVerboseOptionSet(TR_VerbosePerformance,TR_VerboseCodeCache))
+   {
+   if (freePhysicalMemoryB != OMRPORT_MEMINFO_NOT_AVAILABLE)
```

Comment:
I would print this verbose information only if the allocation cannot be made, otherwise it's probably not that useful.

Reply:
Removed this log line from the fix